neurodocker/templates/gentoo.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# Instructions to add NeuroDebian repositories.
+
+name: gentoo
+url: https://www.gentoo.org/
```

Comment:
there is nothing gentoo below yet, misleading commit message etc. Please squash it with whatever actually gentoo specific changes you place into this PR